resources/public/api/conf/2.0/application.raml

```diff
@@ -616,4 +616,77 @@ traits:
                         description: The specified arrival does not exist, has been archived or is not available to the EORI number.
                         value:
                           code: NOT_FOUND
+        /{messageId}:
+          get:
+            displayName: Get a cached message related to a arrival and message ID
+            description: |
+              Retrieve all messages related to a specific arrival and message ID that have been received within the past 31 days.
+
+              For example, this could include ‘Declaration rejected’ or Movement Reference Number (MRN) allocated messages.
```

Comment:
This relates to departures. For arrivals, the P4 equivalent is:

"For example, this could include messages about route diversions or unloading permissions."

Reply:
updated